src/Illuminate/Auth/Console/AuthMakeCommand.php

```diff
@@ -121,6 +121,7 @@ protected function compileControllerStub()
     /**
      * Get full view path relative to the app's configured view path.
      *
+     * @param  $path
```

Comment:
But this misses the type(s); without it, this isn't better then before.

Reply:
edited , thanks to touch  :)